### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by sharmistharanit** [ubsuny/23-Homework3G4#37]
Can you please check the trial1 documentation md file? Is it ok? Then I will work for the final part. @tirthbha

**Comment by tirthbha** [comment]
@sharmistharanit But I think it is not necessary to add trial code in the documentation, right @poojashresthacode ?

**Comment by tirthbha** [comment]
@sharmistharanit Yeah the code is correct but you have to one more libraries that the prof has provided.
